docker/local-metadata/docker-compose-postgres.yml

```diff
@@ -0,0 +1,161 @@
+      OM_AUTH_AIRFLOW_AZURE_AUTHORITY_URL: ${OM_AUTH_AIRFLOW_AZURE_AUTHORITY_URL:-""}
+      OM_AUTH_AIRFLOW_AZURE_SCOPES: ${OM_AUTH_AIRFLOW_AZURE_SCOPES:-[]}
+      OM_AUTH_AIRFLOW_AZURE_CLIENT_ID: ${OM_AUTH_AIRFLOW_AZURE_CLIENT_ID:-""}
+      # OpenMetadata Airflow Google SSO Configuration
+      OM_AUTH_AIRFLOW_GOOGLE_SECRET_KEY_PATH: ${OM_AUTH_AIRFLOW_GOOGLE_SECRET_KEY_PATH:- ""}
+      OM_AUTH_AIRFLOW_GOOGLE_AUDIENCE: ${OM_AUTH_AIRFLOW_GOOGLE_AUDIENCE:-"https://www.googleapis.com/oauth2/v4/token"}
+      # OpenMetadata Airflow Okta SSO Configuration
+      OM_AUTH_AIRFLOW_OKTA_CLIENT_ID: ${OM_AUTH_AIRFLOW_OKTA_CLIENT_ID:-""}
+      OM_AUTH_AIRFLOW_OKTA_ORGANIZATION_URL: ${OM_AUTH_AIRFLOW_OKTA_ORGANIZATION_URL:-""}
+      OM_AUTH_AIRFLOW_OKTA_PRIVATE_KEY: ${OM_AUTH_AIRFLOW_OKTA_PRIVATE_KEY:-""}
+      OM_AUTH_AIRFLOW_OKTA_SA_EMAIL: ${OM_AUTH_AIRFLOW_OKTA_SA_EMAIL:-""}
+      OM_AUTH_AIRFLOW_OKTA_SCOPES: ${OM_AUTH_AIRFLOW_OKTA_SCOPES:-[]}
+      # OpenMetadata Airflow Auth0 SSO Configuration
+      OM_AUTH_AIRFLOW_AUTH0_CLIENT_ID: ${OM_AUTH_AIRFLOW_AUTH0_CLIENT_ID:-""}
+      OM_AUTH_AIRFLOW_AUTH0_CLIENT_SECRET: ${OM_AUTH_AIRFLOW_AUTH0_CLIENT_SECRET:-""}
+      OM_AUTH_AIRFLOW_AUTH0_DOMAIN_URL: ${OM_AUTH_AIRFLOW_AUTH0_DOMAIN_URL:-""}
+      # OpenMetadata Airflow Custom OIDC SSO Configuration
+      OM_AUTH_AIRFLOW_CUSTOM_OIDC_CLIENT_ID: ${OM_AUTH_AIRFLOW_CUSTOM_OIDC_CLIENT_ID:-""}
+      OM_AUTH_AIRFLOW_CUSTOM_OIDC_SECRET_KEY: ${OM_AUTH_AIRFLOW_CUSTOM_OIDC_SECRET_KEY:-""}
+      OM_AUTH_AIRFLOW_CUSTOM_OIDC_TOKEN_ENDPOINT_URL: ${OM_AUTH_AIRFLOW_CUSTOM_OIDC_TOKEN_ENDPOINT_URL:-""}
+      DB_DRIVER_CLASS: ${DB_DRIVER_CLASS:-org.postgresql.Driver}
+      DB_SCHEME: ${DB_SCHEME:-postgresql}
+      DB_USE_SSL: ${DB_USE_SSL:-false}
+      MYSQL_USER: ${MYSQL_USER:-openmetadata_user}
+      MYSQL_USER_PASSWORD: ${MYSQL_USER_PASSWORD:-openmetadata_password}
+      MYSQL_HOST: ${MYSQL_HOST:-postgres}
+      MYSQL_PORT: ${MYSQL_PORT:-5432}
+      MYSQL_DATABASE: ${MYSQL_DATABASE:-openmetadata_db} 
+    expose:
+      - 8585
+      - 8586
+      - 9200
+      - 9300
+      - 5432
+    ports:
+      - 8585:8585
+      - 8586:8586
+    depends_on:
+      - elasticsearch
+      - postgres
+      - ingestion
+    networks:
+      local_app_net:
+        ipv4_address: 172.16.239.13
+    extra_hosts:
+      - "postgres:172.16.239.10"
+      - "elasticsearch:172.16.239.11"
+
+  ingestion:
+    build:
+      context: ../../.
+      dockerfile: ingestion/Dockerfile
+      args:
+        INGESTION_DEPENDENCY: ${INGESTION_DEPENDENCY:-all}
+    container_name: openmetadata_ingestion
+    depends_on:
+      - postgres
+      - elasticsearch
+    expose:
+      - 8080
+    ports:
+      - 8080:8080
+    networks:
+      - local_app_net
+    extra_hosts:
+      - "localhost:172.16.239.10"
+      - "localhost:172.16.239.11"
+      - "localhost:172.16.239.13"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
this is not required anymore! we can get rid of this! cc - @pmbrull @ulixius9

Reply:
Done